misc/python/materialize/checks/all_checks/alter_table.py

```diff
@@ -11,13 +11,12 @@
 from materialize.checks.actions import Testdrive
 from materialize.checks.checks import Check
 from materialize.checks.executors import Executor
+from materialize.mz_version import MzVersion
 
 
 class AlterTableAddColumn(Check):
     def _can_run(self, e: Executor) -> bool:
-        # TODO: Reenable when database-issues#8952 is fixed
-        return False
-        # return self.base_version >= MzVersion.parse_mz("v0.131.0-dev")
+        return self.base_version >= MzVersion.parse_mz("v0.131.0-dev")
```

Comment:
Do we need to limit this to `v0.134.0-dev` or greater, since the fix just landed?

Reply:
True